metricbeat/metricbeat.reference.yml

```diff
@@ -1267,3 +1267,19 @@ logging.files:
 
 # Set to true to log messages in json format.
 #logging.json: false
+
+
+#================================ HTTP Endpoint ======================================
+# Each beat can expose internal data points through a http endpoint. For security
+# reason the endpoint is disabled by default. This feature is currently experimental.
+# Stats can be access through http://localhost:5066/stats For a pretty json output,
+# add ?pretty .
+
+# Defines if http endpoint is enabled
+#http.enabled: false
+
+# Host to expose the http endpoint to. It is recommended to use only localhost.
```

Comment:
The HTTP endpoint will bind to this hostname or IP address.

Reply:
done